src/main/webapp/resources/celJS/celpresentation/CelementsSlideShow.js

```diff
@@ -335,216 +325,216 @@ window.CELEMENTS.presentation.SlideShow = function(containerId) {
-                  fullName, ' and cel_slideShow:nonSlideClick event not handled');
+                fullName, ' and cel_slideShow:nonSlideClick event not handled');
             }
           }
         }
       },
 
-      _registerDirectLinks : function() {
-        var _me = this;
-        _me._htmlContainer.select('.celPresSlideShow_navigation').each(function(navElem) {
-          navElem.select('a').each(function(navLink) {
+      _registerDirectLinks: function () {
+        const _me = this;
+        _me._htmlContainer.select('.celPresSlideShow_navigation').each(function (navElem) {
+          navElem.select('a').each(function (navLink) {
             navLink.stopObserving('click', _me._gotoSlideClickHandlerBind);
             navLink.observe('click', _me._gotoSlideClickHandlerBind);
           });
         });
       },
 
-      setAutoresize : function(doAutoResize) {
-        var _me = this;
+      setAutoresize: function (doAutoResize) {
+        const _me = this;
         _me._autoresize = doAutoResize;
       },
 
-      setOverwritePageLayout : function(overwritePageLayout) {
-        var _me = this;
+      setOverwritePageLayout: function (overwritePageLayout) {
+        const _me = this;
         _me._overwritePageLayout = overwritePageLayout;
       },
 
-      setPreloadSlideAjaxMode : function(preloadSlideAjaxMode) {
-        var _me = this;
-//        console.log('setPreloadSlideAjaxMode: ', preloadSlideAjaxMode);
+      setPreloadSlideAjaxMode: function (preloadSlideAjaxMode) {
+        const _me = this;
+        //        console.log('setPreloadSlideAjaxMode: ', preloadSlideAjaxMode);
         _me._preloadSlideAjaxMode = preloadSlideAjaxMode;
       },
 
-      addPreloadSlideParam : function(params) {
-        var _me = this;
-//        console.log('addPreloadSlideParam: ', params);
+      addPreloadSlideParam: function (params) {
+        const _me = this;
+        //        console.log('addPreloadSlideParam: ', params);
         _me._preloadSlideParams = _me._preloadSlideParams.merge(params);
       },
 
-      setResizeSlide : function(isResizeSlide) {
-        var _me = this;
+      setResizeSlide: function (isResizeSlide) {
+        const _me = this;
         _me._resizeSlide = isResizeSlide;
       },
 
-      setCenterSlide : function(isCenterSlide) {
-        var _me = this;
+      setCenterSlide: function (isCenterSlide) {
+        const _me = this;
         _me._centerSlide = isCenterSlide;
       },
 
-      _preloadSlide : function(slideFN, callbackFN) {
-        var _me = this;
-        callbackFN = callbackFN || function(newSlideContent) {};
-        var params = {
-          'xpage' : 'celements_ajax',
-          'ajax_mode' : 'pageTypeWithLayout',
-          'ajax' : '1'
+      _preloadSlide: function (slideFN, callbackFN) {
+        const _me = this;
+        callbackFN = callbackFN || function (newSlideContent) { };
+        let params = {
```

Comment:
`params` is never reassigned
```suggestion
        const params = {
```

Reply:
Not true. See line 549